src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -1,46 +1,88 @@
 package cromwell.engine
 
+import java.util.UUID
+import java.util.concurrent.Executors
+
 import akka.actor.{Actor, Props}
 import akka.event.{Logging, LoggingReceive}
 import cromwell.binding._
 import cromwell.binding.values.WdlValue
-import cromwell.engine
-import cromwell.engine.backend.Backend
+import cromwell.engine.backend.{TaskAbortedException, Backend}
 import cromwell.engine.workflow.WorkflowActor
 import cromwell.engine.workflow.WorkflowActor.CallFailed
 
+import scala.collection.mutable
+import scala.concurrent.{ExecutionContext, Future}
 import scala.language.postfixOps
-import scala.util.{Failure, Success}
-
+import scala.util.{Try, Failure, Success}
 
 object CallActor {
+  type CallExecutionID = UUID
```

Comment:
There's already DSDEEPB-796 but we should probably curtail adding new `type` aliases in favor of case classes